modules/docker-mirror/startup-script.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+ln -s /dev/disk/by-id/google-registry-data /dev/sdh
```

Comment:
![image](https://user-images.githubusercontent.com/18282288/140810591-937d61f4-ab88-44a0-bf4e-4f6e23484b06.png)

Reply:
Thank you, Bill

Reply:
On a serious note, is there a reason to make this symlink instead of using /dev/disk/by-id/google-registry-data directly?

Reply:
It was easier to keep the scripts in sync between google and aws if only that part differs.